_posts/2016/06/2016-06-31-brazil.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+authors: ["Raniere Silva"]
+title: "Three workshops in Brazil"
+date: 2016-06-31
+time: "10:00:00"
+category: ["workshops"]
+---
+In Brazil, like in many other countries that we are starting to run workshops,
+there are many open questions that we need to answer such as
+
+- how do we advertize our workshops?
+- do we charge a registration fee? If yes, how much do we charge?
+- how do we create a local community after the workshop?
+- how do we get new partners after create a local community?
+
+Thanks to the work of many instructors like
+Anelda van der Walt,
+Belinda Weaver,
+Bill Mills,
+Lex Nederbragt,
+Selene L. Fernandez-Valverde, and
+Tiffany Timbers
+we are getting some hypothesis to the questions above across the world.
+In terms of Brazil, the three workshops that we ran in May added more data
+to help answering the questions.
+
+## Workshop in Florianópolis
+
+[This](https://rgaiacs.github.io/2016-05-16-scipyla/) was the [second](https://dbarneche.github.io/2014-12-11-ufsc/)
+workshop that we run in Florianópolis
+with more than one year between each.
+
+Our 20 seats sold out in a few days (we didn't charge anything) but only four students attended all the sessions making this the workshop in Brazil with the smallest number of attendees. The low number of attendees can be due our workshop ran at the same time of others activities of [SciPy Latin America 2016](http://scipyla.org/conf/2016/) but I believe that the main reason was the missing of a local champion that motivate people to attend the workshop (in 2014 we had Diego Barneche Rosado).
```

Comment:
some rewording on this line s/ran/running ;  s/time of/time as/  ; s/champion that/champion to/